### DISCUSSION TITLE
No documents are valid to register in GitHub students

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21992] [New to GitHub]
<p>Hi!<br>
I’m trying to register in Github Student Program. I’m 15 and I frequent an Italian School. I don’t know wich document I can to send you because it doesn’t exist a current enrollment document. I sent you a screenshot of my electronic register and of my evaluation document and you don’t accept.<br>
When I upload the document it says me “Your image should include the name of your school” but the school name is visible!</p>
<p>I contact the support but they answered me with a precompiled answer that didn’t help me.<br>
Can you help? Thanks</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Welcome to the Community Forum, <a class="mention" href="/zoiaenrico">@zoiaenrico</a>! For student benefits and related conversations, we recommend using our <a href="https://github.com/orgs/github-community/discussions/categories/github-education">Education Discussion</a> as you’ll have better visibility among those who may be able to answer your question.</p>
